doc/src/learn/exchange-integration-guide.md

```diff
@@ -83,7 +83,9 @@ Use the steps in this section to install and configure a Sui Full node directly
     ```
 ## Set up Sui addresses
 
-Sui addresses do not require on-chain initialization, you can spend from an address if it corresponds to your private key. You can derive a Sui address by hashing the signature scheme flag byte concatenated with public key bytes `flag || pubkey` using the [BLAKE2b](https://www.blake2.net/) (256 bits output) hashing function. The following code sample demonstrates how to derive a Sui address in Rust:
+Sui addresses do not require on-chain initialization, you can spend from an address if it corresponds to your private key. You can derive a Sui address by taking the first 32 bytes of the result by hashing the signature scheme flag byte concatenated with public key bytes `flag || pubkey` using the [BLAKE2b](https://www.blake2.net/) (256 bits output) hashing function. Currently, Sui address supports these signature schemes: pure Ed25519, Secp256k1, Secp256r1 and Multisig. 
```

Comment:
> taking the first 32 bytes of the result
well with blake2b_256 the output is 32 bytes, no need to truncate any more, we can remove the part and say we use the blake2b_256 output (no truncation)